### DISCUSSION TITLE
Energy dashboard improvements, e.g. zoom and use entire page

### DISCUSSION THREAD

**Discussion by kranendijk** [home-assistant/frontend#18571] [Dashboard] [answered]
Since last release the energy dashboard is not filling the entire page on landscape mode. If the cards would fill up to the size of the screen they would be better readable I think.
Furthermore: it would be great if an individual card (electricity, distribution) could be “zoomed” like you do on a webpage or a picture on a tablet or phone. I mean by using your fingers. I do not know the exact term for that 🙃
Also the gauges indicating used solar, percentage used from net and so on do not necessarily need to be there. An option to show or hide them would be great.
Not asking for myself, but trying to get my wife being entiousiastic for HA. Call it the WAF, which allows me to buy more goodies to integrate.

In short my request could be interpreted as “make the energy dashboard customizable”.

Happy to test or elaborate!

Eric

Some pictures:
![IMG_0328](https://github.com/home-assistant/frontend/assets/53447746/eb8f4c6f-e074-424c-ab15-1cd87fb0a8b6)
![IMG_0329](https://github.com/home-assistant/frontend/assets/53447746/37a50fa3-b225-4c4d-bc64-abf028e761af)

**Comment by karwosts** [comment, marked as answer]
The empty space is caused by a card-mod issue, try updating that resource to fix it

**Reply by kranendijk** [reply]
Yes that worked, now the rest remains a feature request I suppose?

**Comment by dougiteixeira** [comment]
> In short my request could be interpreted as “make the energy dashboard customizable”.

You can build your personalized dashboard with energy cards, see how to add the cards in [this documentation](https://www.home-assistant.io/dashboards/energy/).
